pkg/ocm/client.go

```diff
@@ -89,6 +90,7 @@ func (b *ClientBuilder) Build() (result *Client, err error) {
 	// values in the configuration, so that default values won't be overridden:
 	builder := sdk.NewConnectionBuilder()
 	builder.Logger(logger)
+	builder.Agent(info.UserAgent + "/" + info.Version)
```

Comment:
FYI, this overrides the existing user agent (which is currently `OCM-SDK/0.1.235`). Not sure this is an issue, just wanted to call it out in case we use anything that relies on it.

Ideally you want to prepend it so that it ends up as `ROSACLI/1.1.7 OCM-SDK/0.1.235`

Reply:
I think overriding isn't an issue, and prepending it doesn't hurt. You can do like this:

```go
builder.Agent(info.UserAgent, + "/" + info.Version + " " + sdk.DefaultAgent)
```